# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../amarok/mysql.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur amarok-2.0.90-orig/cmake/modules/FindMySQLAmarok.cmake amarok-2.0.90/cmake/modules/FindMySQLAmarok.cmake
--- amarok-2.0.90-orig/cmake/modules/FindMySQLAmarok.cmake	2009-04-21 13:41:47.000000000 +0300
+++ amarok-2.0.90/cmake/modules/FindMySQLAmarok.cmake	2009-04-21 13:55:03.000000000 +0300
@@ -16,7 +16,7 @@
 find_path(MYSQL_INCLUDE_DIR mysql.h
    ~/usr/include/mysql
    /opt/local/include/mysql5/mysql
-   /opt/mysqle/include/mysql
+   /opt/mysql/include/mysql
    /opt/mysql/mysql/include 
    /usr/mysql/include/mysql
    /usr/include/mysql
@@ -44,7 +44,7 @@
    PATHS
    ~/usr/lib/mysql
    /opt/local/lib/mysql5/mysql
-   /opt/mysqle/lib/mysql
+   /opt/mysql/lib/mysql
    /usr/lib/mysql
    /usr/lib64/mysql
    /usr/local/lib/mysql
